File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 8/1/2025, 10:42:21 PM
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count=0
        for i in nums:
            if val==i:
                count+=1
        for i in range(1,count+1):
            nums.remove(val)
        return len(nums)    

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 8/1/2025, 10:42:12 PM
class Solution {
    public int removeElement(int[] nums, int val) {
     ArrayList<Integer> arr=new ArrayList<>();
     int c=0;
     for(int i=0;i<nums.length;i++){
        if( nums[i]!=val){
        arr.add(nums[i]);
        c++;
     }
     }
      for(int i=0;i<arr.size();i++){
        nums[i]=arr.get(i);
      }
return c;
    }
}